schema/bom-1.6.proto

```diff
@@ -1150,9 +1152,136 @@ message ModelCard {
       // With respect to the benefits and harms outlined, please describe any mitigation strategy implemented.
       optional string mitigationStrategy = 4;
     }
+    message EnvironmentalConsiderations {
+      // Describes energy consumption information incurred for one or more component lifecycle activities.
+      repeated EnergyConsumption energyConsumptions = 1;
+    }
+    message EnergyConsumption {
+      // Describes energy consumption information incurred for the specified lifecycle activity.
+      ActivityType activity = 1;
+      // The providers of the energy consumed by the associated model development lifecycle activity.
+      repeated EnergyProviderType energySources = 2;
+      // The total energy cost associated with the model lifecycle activity.
+      EnergyMeasureType energyCost = 3;
+      // The CO2 cost or debit equivalent to the total energy cost.
+      optional CO2MeasureType co2CostEquivalent = 4;
+      // The CO2 offset or credit for the CO2 equivalent cost.
+      optional CO2MeasureType co2CostOffset = 5;
+    }
   }
 }
 
+// An activity that is part of a machine learning model development or operational lifecycle.
+enum ActivityType {
+  // a lifecycle activity type whose description does not match currently defined values.
+  // buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX -- `other` is our fallback
+  ACTIVITY_TYPE_OTHER = 0;
```

Comment:
pleasse do not use `ACTIVITY_TYPE_OTHER ` as the default value.
better create `ACTIVITY_TYPE_UNSPECIFIED` for that purpose.\


the things where you have seen the `  // buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX -- `other` is our fallback` were added to suppress the messache in acknowledgement of a mistake in the past.

Reply:
Hey Jan, so a receiver of this message would have to map "unspecified" to "other"?  or should we actually define an enum value unspecified for all new enum. types (i.e., an actual value "unspecified" in the JSON and XML?

Reply:
"other" is a well-known case, according to JSON and XML.

But the value `0`, in Brotobuf v3(`proto3` schema), is the default value which is assigned if no value is given (for optional messages for example) and it is also the value that is assigned if a non-wellnown-value is received.
the case `...UNSPECIFIED = 0` is a protobuf3 thing only - no need to carry this design over to XML/JSON.

Reply:
read https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum

Reply:
all done and fixed :+1: